SegmentEditorFloodFilling/SegmentEditorFloodFillingLib/SegmentEditorEffect.py

```diff
@@ -123,10 +125,14 @@ def updateGUIFromMRML(self):
     self.neighborhoodSizeMmSlider.blockSignals(True)
     self.neighborhoodSizeMmSlider.value = self.scriptedEffect.doubleParameter("NeighborhoodSizeMm")
     self.neighborhoodSizeMmSlider.blockSignals(False)
+    self.roiSelector.blockSignals(True)
+    self.roiSelector.setCurrentNode(self.scriptedEffect.parameterSetNode().GetNodeReference("ROINodeID"))
+    self.roiSelector.blockSignals(False)
```

Comment:
I saw that at a few places this was messed up and probably you were just unlucky to copy-paste from there. The proper way to block/unblock signals is:
```
wasBlocked = self.something.blockSignals(True)
...
self.something.blockSignals(wasBlocked) 
```

Using `self.something.blockSignals(False)` would always end up with _unblocking signals_, while you what you wanted was to _temporarily block the signals_.